client/my-sites/activity/activity-log/index.jsx

```diff
@@ -565,7 +565,7 @@ export default connect(
 		const siteIsOnFreePlan =
 			isFreePlan( get( getCurrentPlan( state, siteId ), 'productSlug' ) ) &&
 			! isVipSite( state, siteId );
-		const siteHasNoLog = siteIsOnFreePlan && ! siteHasBackupProductPurchase( state, siteId );
+		const siteHasNoLog = siteIsOnFreePlan && ! siteHasJetpackProductPurchase( state, siteId );
```

Comment:
If `siteHasJetpackProductPurchase()` includes more than just backup and scan plans then we could instead do:

`const siteHasNoLog = siteIsOnFreePlan && ! siteHasBackupProductPurchase( state, siteId ) && ! siteHasScanProductPurchase( state, siteId );`

Reply:
I think I agree with @jwidavid here. `siteHasJetpackProductPurchase` would also include sites with Search, which I don't believe includes Activity Log access (does it?)

Reply:
It does not. This also would include Jetpack Anti-Spam, which I know doesn't have this. Maybe it would make more sense to have a new selector specifically for products that have AL access? We do similar logic elsewhere.

Reply:
> It does not. This also would include Jetpack Anti-Spam, which I know doesn't have this. Maybe it would make more sense to have a new selector specifically for products that have AL access? We do similar logic elsewhere.

That may be a good idea, but perhaps for a separate task? My knowledge of Calypso is fairly limited, but if this is the only place that it would likely be used then (at least for this task) it would probably be better to just add the extra condition? The sooner scan only plans get AL access the better.

Reply:
@jwidavid - That makes sense! I'd suggest we can do a quick solution now by checking if the site has _any_ Jetpack plan, and doing a check of Backup or Scan products via getting the product slugs and comparing to an array. We honestly need some better utilities of "does this site have access to this feature" anyway.

Reply:
@ChaosExAnima the changes I've made check for Backup and Scan purchased products, if no purchased products are found, then the AL is not provided. This has the desired effect. Would you like to see the logic changed here?

Otherwise - I'll wait for someone from Voyager to approve the PR 👍

Reply:
Changes look good! Didn't see those 🙂